kategory/src/test/kotlin/kategory/laws/Law.kt

```diff
@@ -2,3 +2,5 @@ package kategory
 
 data class Law(val name: String, val test: () -> Unit)
 
+inline fun <reified A> A.equalUnderTheLaw(b: A, eq: Eq<A> = Eq()): Boolean =
```

Comment:
Can we have extension functions that are `inline reify... <F, ..>` Once we have instances for the datatypes as part of our effort to provide syntax all datatypes will include syntax for `eqv` and `neqv` which can be used directly here provided thre is a registeres `Eq` instance for the datatype being compared.